File: sem2_c_cpp_java/Assignment6_14_02_23/SampleCodes_Inheritance_Costructor_CPP_14_2_23/CPP/SLI1.cpp
#include<iostream>
using namespace std;

class A
{
	public:
		void f1()
		{
			cout << "HEllo" << endl;
		}
};

class B : public A
{
	public:
		void f2()
		{
			cout << "Hi" << endl;

		}
};

int main()
{
	class A a1;
	class B b1;
	a1.f1();
	b1.f2();
}

File: sem2_c_cpp_java/Assignment6_14_02_23/SampleCodes_Inheritance_Java_14_2_23/Java/SLI1.java
import java.util.*;

class A1
{
	public void f1()
	{
		System.out.println("Hello");
	}

	public static void main(String args[])
	{
		B1 b1 = new B1();
		b1.f1();
		b1.f2();
	}
};

class B1 extends A1
	{
		public void f2()
		{
			System.out.println("Hello from B1");
		}
	};


